specs/merge/beacon-chain.md

```diff
@@ -223,6 +245,62 @@ def compute_timestamp_at_slot(state: BeaconState, slot: Slot) -> uint64:
     return uint64(state.genesis_time + slots_since_genesis * SECONDS_PER_SLOT)
 ```
 
+### Beacon state accessors
```

Comment:
Altair is frozen, but it might make sense to go for code-reuse anyway by not copying it over. That said, I'm also kind of happy we can easily extract a version of the spec that doesn't contain any legacy preset/config options, for new client implementations to join.

Reply:
by instead providing a config getter function -- e.g. `get_inactivity_penalty_quotient(epoch)` or something?

yeah, I see the argument on both sides ..